packages/extensions-registry/package.json

```
@@ -0,0 +1,46 @@
+{
+	"name": "@directus/extensions-registry",
+	"version": "0.0.0",
```

```suggestion
	"version": "0.1.0",
```

Not sure if its important but wanted to mention just in case

---

Looks strange but `0.0.0` is indeed correct, as it will be bumped by changeset 👍